tests/unit/lms/services/exceptions_test.py

```diff
@@ -141,32 +141,26 @@ def test_status_code(self, response, expected):
         assert err.status_code == expected
 
     @pytest.mark.parametrize(
-        "message,request_,response,cause,expected",
+        "response,cause,expected",
         [
             (
                 None,
                 None,
-                None,
-                None,
-                "ExternalAsyncRequestError(message=None, cause=None, request=Request(method=None, url=None, body=None), response=Response(status_code=None, reason=None, body=None), validation_errors=None)",
+                "ExternalAsyncRequestError(cause=None, request=Request(method=None, url=None, body=None), response=Response(status_code=None, reason=None, body=None))",
             ),
             (
-                "Some message",
-                None,
                 Mock(
-                    status=400, reason="OK", request_info=Mock(method="POST", url="URL")
+                    status=400,
+                    reason="OK",
+                    request_info=Mock(method="POST", url="URL"),
                 ),
                 KeyError("cause"),
-                "ExternalAsyncRequestError(message='Some message', cause=KeyError('cause'), request=Request(method='POST', url='URL', body=None), response=Response(status_code=400, reason='OK', body=None), validation_errors=None)",
+                "ExternalAsyncRequestError(cause=KeyError('cause'), request=Request(method='POST', url='URL', body=None), response=Response(status_code=400, reason='OK', body=None))",
```

Comment:
This is the async version of the test. Removed the deleted attributes.

The sync version code/test didn't change.